src/content/vintage/getting-started/observability/logging/architecture/index.md

```diff
@@ -0,0 +1,72 @@
+user_questions:
+  - What is the logging architecture?
+  - Why is Giant Swarm using loki?
+  - Why is Giant Swarm recommending loki?
+  - Which logs are stored by Giant Swarm?
+  - Where are the logs stored by Giant Swarm?
+aliases:
+  - /getting-started/observability/logging/architecture
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2024-03-21
+---
+
+Logging is an important pillar of observability and it is thus only natural that Giant Swarm provides and manages a logging solution for operational purposes.
+
+This document gives an overview of how logging is managed by Giant Swarm, including which logs are stored, which tools we use to ship and store them, as well as why we chose those tools in the first place.
+
+## Overview of the logging platform
+
+Here is an architecture diagram of our current logging platform:
+
+![Logging pipeline architecture overview](logging-architecture.png)
+<!-- Source: https://drive.google.com/file/d/1Gzl0mTdJcaui_zIC9QuHcgMX3QJygALo -->
+
+In this diagram, you can see that we run the following tools in each management cluster as part of our logging platform:
+
+- `Grafana Loki` that is accessible through our managed Grafana instance.
+- `multi-tenant-proxy`, a proxy component used to handle multi-tenancy for Loki.
+- A couple of logging agents (`Grafana Promtail` and `Grafana Agent`) that run on the management cluster and your workload clusters alike. We currently need two different tools for different purposes.
+    - Promtail is used to retrieve the container and kubernetes audit logs
+    - Grafana Agent is used to retrieve the kubernetes events.
+
+If you want to play with Loki, you should definitely check out our guides explaining [how to access Grafana]({{< relref "/vintage/getting-started/observability/visualization/access" >}}) and how to [explore logs with LogQL]({{< relref "/vintage/getting-started/observability/visualization/log-exploration" >}})
+
+## Logs stored by Giant Swarm
+
+Kubernetes clusters produce a vast amount of machine and container logs.
+
+The logging agents that we have deployed on management and workload clusters currently send the following logs to Loki:
+
+- Kubernetes Pod logs from the `kube-system` and `giantswarm` namespaces.
+- Kubernetes Events created in the `kube-system` and `giantswarm` namespaces.
+- [Kubernetes audit logs]({{< relref "./audit-logs#kubernetes-audit-logs" >}})
+
+In the future, we will also store the following logs:
+
+- [Machine (Node) audit logs]({{< relref "./audit-logs#machine-audit-logs" >}})
+- Teleport audit logs, tracked in https://github.com/giantswarm/roadmap/issues/3250
+- Giant Swarm customer workload logs as part of our observability platform, tracked in https://github.com/giantswarm/roadmap/issues/2771
+
+## Why we prefer Loki over its competitors
+
+The reasons that lead us to using Grafana Loki in favor of its competitors (which boils down to only opendistro in our case) are numerous.
```

Comment:
```suggestion
There are numerous reasons to use Grafana Loki in favor of its competitors.
```
Just less cooking here, because we already use that expression and also mention OpenDistro below.